src/App.css

```diff
@@ -1,32 +1,258 @@
-.verticalLine {
-  border-right: thick solid #030000;
+@import url('https://kit.fontawesome.com/a076d05399.js');
+
+.Bookstore-CMS {
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+.user {
+  margin-left: 25%;
+  padding: 0.875rem;
+  border: 1px solid #e8e8e8;
+  border-radius: 50%;
+}
+
+.fa-user {
+  color: #0290ff;
+}
+
+.panel-bg {
+  display: flex;
+  gap: 30px;
+  font-family: 'Montserrat', sans-serif;
+  align-items: center;
+  width: 87.5rem;
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  background-color: var(--white-two);
+}
+
+.navlink {
+  text-decoration: none;
+  color: #121212;
+  opacity: 0.5;
+}
+
+.BOOKS,
+.categories {
+  text-transform: uppercase;
+  font-size: 0.813rem;
+  letter-spacing: 1.9px;
+}
+
+.Line {
+  width: 100%;
+  margin: 2.5rem 4.063rem 1.813rem 0;
+  border: solid 1px #e8e8e8;
+}
+
+.Title {
+  width: 10.875rem;
+  height: 1.5rem;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+  margin-bottom: 2rem;
 }
 
-nav ul {
+/*  book style starts here */
+.container {
```

Comment:
- [ ] Most of the texts in this section use `RobotoSlab` as a font family. Kindly import it from Google fonts and use it to match the design. 🧐️ 
- [ ] Please put the text `CURRENT CHAPTER` on a single line. ✍️ 
- [ ] Kindly adjust the spacing between the left section and the circle. Currently, there is a bit much whitespace on the right side. 🤓️
- [ ] The separator is between the status and the right section is a single line. Kindly make it as per the design. 

**Your design**
![Screenshot from 2023-03-11 12-42-40](https://user-images.githubusercontent.com/17243325/224478507-3ac06628-0ddd-4640-9384-76ce9a696cd2.png)

**Zeplin design**
![Screenshot from 2023-03-11 12-42-13](https://user-images.githubusercontent.com/17243325/224478506-7c72f0e2-cefd-4850-8a96-834cd7d46094.png)

Reply:
thank you @amejid , i have implemented all your requirements.

Reply:
@amejid , "the current chapter" is on a single line. Kindly, check your browser. 
<img width="265" alt="Screenshot 2023-03-11 at 14 05 30" src="https://user-images.githubusercontent.com/43499747/224483402-db05d644-0d42-4307-b0df-a901f4b8e868.png">